core/google/api/core/retry.py

```diff
@@ -77,7 +125,8 @@ def exponential_sleep_generator(
             be greater than 0.
         maximum (float): The maximum about of time to delay.
         multiplier (float): The multiplier applied to the delay.
-        jitter (float): The maximum about of randomness to apply to the delay.
+        max_jitter (float): The maximum about of randomness to apply to the
```

Comment:
I have in the past relied on [this article](https://www.awsarchitectureblog.com/2015/03/backoff.html) which, after empirical testing, suggests that the correct way to do jitter is, counter-intuitively, `random.uniform(0, delay)`.

Stopping short of claiming that you must do it this way, but it is what we have done elsewhere.